### ISSUE TITLE
A larger app example

### ISSUE THREAD

**Issue by paulirish** [samccone/The-cost-of-transpiling-es2015-in-2016#43]
A few conversations have concluded that this project would benefit from also testing an app that's larger than TodoMVC. 

We are looking for an opensource webapp authored in vanilla ES6 with ES6 modules.

If anyone has any suggestions, please drop a comment.

**Comment by paulirish** [comment]
cc @guybedford - I feel like if anything, the SystemJS community may have a good candidate or two.

**Comment by nstepien** [comment]
This has potential to become the h5bp of ES2015+.

**Comment by samccone** [comment]
@nolanlawson do you know off of the top of your head if pouchdb would work? But that might not be what we want since it is not a real "app"

**Comment by nolanlawson** [comment]
Yeah, PouchDB is not really an "app" per se, so may be a bad choice. However, I'd be happy to convert [pokedex.org's code](https://github.com/nolanlawson/pokedex.org) to ES6 modules and test it out. A tool like [cjs-to-es6](https://github.com/nolanlawson/cjs-to-es6) could also potentially be used to upgrade just about any CommonJS codebase.

**Comment by samccone** [comment]
pokedex would be a pretty good candidate.. The Marionette v3 branch is all written with es6... but again a lib not really an app.

I think we would have to do quite a bit of work to get pokedex to actually work with property mangling.. but... willing to help :) 

https://github.com/samccone/The-cost-of-transpiling-es2015-in-2016/commit/cbe39967390aae0b4b9c3eae2229465ababb6fcb

**Comment by paulfalgout** [comment]
I definitely see value in trying a larger project that includes some external dependencies, but I think we should leave property mangling out of it. Granted I'm sure it's something you could consider from a library perspective, but that seems like a lofty requirement for any sort of standard project, and so I think it just skews the results away from most people's reality.

**Comment by elibdev** [comment]
The way I understand it, one of the awesomest things about tree shaking / dead code elimination is that you can import just a few functions from a library and instead of including the whole dependency, the compiler will include only the functions you imported. 

This is a big win for the simple reason that you can include a large library as a dependency, and only the parts of it that you use will end up in the compiled Javascript. 

If one of the intentions of including a larger app is to demonstrate this ability, maybe even just adding some library dependency to the TodoMVC app and importing one function from the library would do it. Any build process that use tree shaking will include only that one function in the compiled output, while build processes that don't will just include the whole thing. For a big dependency, the difference should be dramatic.
